.github/workflows/grass-manual.yml

```
@@ -0,0 +1,72 @@
+name: GRASS GIS addon manual
+
+on: [push]
+  # push:
+  #   branches: [ GRASSmanual ]
+
+jobs:
+  build-grass-manual:
+    name: build-grass-manual
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: awalsh128/cache-apt-pkgs-action@latest
+        with:
+          packages: grass grass-dev grass-doc wget make
+          # version: 1.0
+      - name: Install dependencies
+        run: |
+          # apt-get install -y -qq grass grass-dev grass-doc wget make
+          mkdir public
+          # some files are not installed properly, get them online
+          (cd public && wget https://grass.osgeo.org/grass-stable/manuals/grass_logo.png)
+          (cd public && wget https://grass.osgeo.org/grass-stable/manuals/grassdocs.css)
+      - name: Create a temporary location (needed for v<7.6)
+        run: |
+          grass -c EPSG:4326 grasstmploc -e
+      - name: Compile GRASS GIS addon
+        run: |
+          ADDON_NAME=$(echo ${GITHUB_REPOSITORY} | cut -d "/" -f 2)
+          echo ${ADDON_NAME}
+          # expecting name of module is repository name
+          grass grasstmploc/PERMANENT --exec g.extension extension=${ADDON_NAME} url=. prefix=build --verbose
+      - name: Move manual to public folder
+        run: |
+          ADDON_NAME=$(echo ${GITHUB_REPOSITORY} | cut -d "/" -f 2)
+          # put the compile documentation to the public directory
+          # public name is convention for published web pages
+          cp -r build/docs/html/* public/
+          cp public/${ADDON_NAME}.html public/index.html
+      - name: Reset URL
+        run: |
+          sed -i "s+file:///usr/lib/grass78/docs/html/+https://grass.osgeo.org/grass83/manuals/+g" public/index.html
```

```suggestion
          sed -i "s+file:///usr/lib/grass78/docs/html/+https://grass.osgeo.org/grass-stable/manuals/+g" public/index.html
```

Please consider to use `-stable` to point to G8, this always works, also once G8.4 is (soonish) out.